src/App/Lobby/LobbyService.ts

```diff
@@ -0,0 +1,43 @@
+import AddressHelper from '../AddressHelper';
+import request from 'superagent';
+
+export interface IPlayerInRoom {
+  ID: number;
+  name: string;
+  credentials?: string;
```

Comment:
this is a single credential or more than one? why credentialS ?

Reply:
One, made singular

Reply:
I did this because BGIO returns 'playerCredentials' -- I think I agree that credential makes more sense, though.

https://github.com/freeboardgame/FreeBoardGame.org/blob/cbb89332e46a8373c415a921cc43149302294982/src/App/Lobby/LobbyService.ts#L41